flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -194,8 +194,8 @@ run_test "Stateful stream job upgrade end-to-end test" "$END_TO_END_DIR/test-scr
 
 run_test "Netty shuffle direct memory consumption end-to-end test" "$END_TO_END_DIR/test-scripts/test_netty_shuffle_memory_control.sh"
 
-run_test "Elasticsearch (v5.3.3) sink end-to-end test" "$END_TO_END_DIR/test-scripts/test_streaming_elasticsearch.sh 5 https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.3.3.tar.gz"
-run_test "Elasticsearch (v6.3.1) sink end-to-end test" "$END_TO_END_DIR/test-scripts/test_streaming_elasticsearch.sh 6 https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.3.1.tar.gz"
+run_test "Elasticsearch (v5.3.3) sink end-to-end test" "$END_TO_END_DIR/test-scripts/test_streaming_elasticsearch.sh 5 http://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.3.3.tar.gz"
```

Comment:
Why's the `http` version more stable than `https`?

Reply:
80% of all errors are TLS errors, we don't necessarily need TLS if it makes the downloads more stable

Reply:
Thanks! This PR should be good to go then.